src/main/webapp/app/exercises/text/assess-new/textblock-feedback-editor/textblock-feedback-editor.component.ts

```diff
@@ -69,4 +69,26 @@ export class TextblockFeedbackEditorComponent implements AfterViewInit {
     didChange(): void {
         this.feedbackChange.emit(this.feedback);
     }
+    /**
+     * Allows the drop of an SGI Element
+     */
+    allowDrop(event: DragEvent) {
+        event.preventDefault();
+    }
```

Comment:
I think this method should use a better name. Currently, I would prefer `$event.preventDefault()` in the template.